packages/app/src/components/HelpModal/HelpModal.tsx

```diff
@@ -128,8 +130,15 @@ function HelpModal(props: HelpModalProps) {
           <Show when={gpuDeviceInfo()} keyed>
             {(deviceInfo) => (
               <>
-                <p>Adapter: {deviceInfo.displayName}</p>
+                {deviceInfo.description !== '' ? (
+                  <p>Name: {deviceInfo.description}</p>
+                ) : undefined}
+                <p>Vendor: {deviceInfo.vendor}</p>
+                <p>Architecture: {deviceInfo.architecture}</p>
```

Comment:
Perhaps fallback all of this to unknown string, or something like that? Or just redact the whole row in that case?